tests/integration/README.md

```diff
@@ -0,0 +1,33 @@
+# How to launch the QEMU test environment
+
+## Requirements
+
+- DPDK
+- Linux kernel source
+- QEMU KVM >= 2.2
+- brctl (from bridge-utils)
+- multistrap
+
+## Compilation
+
+1. Compile a Linux kernel with the provided config file (```config```)
+2. Copy the bzImage file in this folder (integration)
+3. Export the RTE_KERNELDIR environment variable, it should contain the path to the kernel package (for instance: ```/home/pktj/linux-4.1.7/```)
+4. Compile DPDK
+5. Compile pktj with BUILD_TARGET=qemu (```make BUILD_TARGET=qemu```)
+
+## First run
+
+1. Tweak lab.sh as you wish
+2. Export RTE_SDK and RTE_TARGET environment variables (it should be already done if you have compiled pktj successfully)
+3. Run the lab with:
+```
+./lab.sh
+```
+4. Connect to VMs using the output from the previous command
+5. pktj is now ready to start on r2, you can use the sample configuration file in ```/mnt/build/conf/pktj.conf```
```

Comment:
we need the touch /tmp/abc or we may modify pktj.conf for using /dev/null as acl file